### ISSUE TITLE
Missing scripts/data

### ISSUE THREAD

**Issue by jmonlong** [vgteam/sv-genotyping-paper#5]
There are a few links/scripts that are still missing. @glennhickey could you add them?

- SMRTSV v2 commands (ran on courtyard) in the READMEs in `human/svpop` and `human/chmpd`.
- The raw files to make the CHMPD VCF (`download.sh` file missing) in `human/chmpd`.
- The links to the reads for CHMPD in `human/chmpd`.
- The link to the raw GIAB reads and the commands to downsample them to 50x in `human/giab`.

**Comment by glennhickey** [comment]
This should be resolved now.  Why are we using GIAB 0.5 and not 0.6?  This contradicts the manuscript text and doesn't seem to make sense.

**Comment by glennhickey** [comment]
Whatever the case, the good news is that .5 and .6 are very similar.  We just have to make sure the text is consistent with this repo.  If we are indeed using .5, will have to double check the manuscript to make sure it all makes sense.  (ie not sure the parental calls are ./. in .5 like we allude to in the text).

**Comment by jmonlong** [comment]
In the end we had ran the other genotypers on v0.5 in case we wanted to compare with the preprint that used 0.5. As you said the numbers are very similar. I updated the text/numbers. 

In 0.5, some variants have `./.` on the 3 individuals and some others are parental calls where HG002 is `0/0`. I updated the relevant sentence in the methods sections.

This is in [PR 91](https://github.com/jmonlong/manu-vgsv/pull/91)

(Thanks for completing the missing sections!)
